alluxiofs/core.py

```diff
@@ -118,21 +112,16 @@ def __init__(
                     + self.fs.protocol
                 )
         elif target_protocol is not None:
-            self.fs = filesystem(target_protocol, **self.kwargs)
+            self.fs = filesystem(target_protocol, **self.target_options)
             self.target_protocol = target_protocol
         test_options = test_options or {}
         set_log_level(logger, test_options)
         if test_options.get("skip_alluxio") is True:
             self.alluxio = None
         else:
             self.alluxio = AlluxioClient(
-                etcd_hosts=etcd_hosts,
-                worker_hosts=worker_hosts,
-                options=options,
-                concurrency=concurrency,
-                etcd_port=etcd_port,
-                worker_http_port=worker_http_port,
                 test_options=test_options,
```

Comment:
can you put it also in **kwargs
but making sure user can use original ways to pass in all configuration
e.g. all previous etcd_hosts=etcd_hosts and etc